web/src/app/dialogs/new-inspection/components/group-parameter.component.ts

```diff
@@ -104,6 +104,6 @@ export class GroupParameterComponent {
    * Toggle the collapsed status for children.
    */
   toggle() {
-    this.collapsedFromUserInput.set(this.childrenStatus() === 'collapsed');
+    this.collapsedFromUserInput.set(this.childrenStatus() === 'expanded');
```

Comment:
We might want to add comment for `collapsedFromUserInput` to clarify when it's true. 
I initially thought it's true when collapsed but not sure what `FromUserInput` indicates and it seems we changes the behavior here.